.gitignore

```diff
@@ -67,9 +67,20 @@ SwiftScripts/.build-**
 # Local Netlify folder
 .netlify
 
+# Generated js files we don't need committed
+Sources/ApolloCodegenLib/Frontend/dist/
+
 # Local generated packages we don't need in the main project
 Sources/AnimalKingdomAPI/Generated/Package.swift
 Sources/AnimalKingdomAPI/Generated/Package.resolved
-
-# Generated js files we don't need committed
-Sources/ApolloCodegenLib/Frontend/dist/
+Tests/TestCodeGenConfigurations/**/AnimalKingdomAPI/
```

Comment:
This is to ignore all the generated code so it's not accidentally committed in the future.